CONTRIBUTING.md

```diff
@@ -100,6 +100,29 @@ The packages are available at the following urls with hot-reloading:
 - Front-end: http://localhost:3000
 - Back-end: http://localhost:3100
 
+#### Accessing the MongoDB database
+
+To access MongoDB with the [mongosh shell](https://www.mongodb.com/docs/mongodb-shell/), run the following command:
+
+```sh
+docker exec -it mongo bash
+```
+
+Alternatively, if you are using Docker Desktop, you can click the CLI button to execute the shell for the Mongo container.
```

Comment:
Do we want to mention they can use Mongo Compass?

Reply:
Yes, that sounds like a great idea. I didn't even know about Mongo Compass until now. I'll download it and test it out and also add any instructions for that.

Reply:
Compass seems great, I've added some instructions. 

I'm on macOS. I'm assuming the UI on Windows and Linux are similar but I haven't tested them out. If anyone on either of those platforms wants to give it a try, we can adjust the copy for those platforms.